fairing/frameworks/lightgbm.py

```diff
@@ -149,6 +154,12 @@ def execute(config,
         docker_registry: registry to push the built docker image
         base_image: base image to use for this job. It should have lightgbm installed and should be in PATH variable.
         namespace: Kubernetes namespace to use
+        stream_log: True - streams logs from the first worker in the training job after job launch till the training is finished.
```

Comment:
Thanks for documenting these. This should show up in help(lightgbm.execute), correct?

Reply:
yes.